docs/products/clickhouse/howto/connect-with-nodejs.rst

```diff
@@ -0,0 +1,88 @@
+Connect to the Aiven for ClickHouse® service with Node.js
+=========================================================
+
+Learn how to connect to your Aiven for ClickHouse® service with Node.js using the official Node.js client for connecting to ClickHouse and the HTTPS port.
+
+Pre-requisites
+--------------
+
+* `Node.js <https://nodejs.org/en/download/>`_ in your environment
+* `Node.js client for connecting to ClickHouse <https://clickhouse.com/docs/en/integrations/language-clients/nodejs/>`_
+
+.. tip::
+
+    You can install the Node.js client for connecting to ClickHouse using
+
+    .. code:: java
+
+        npm i @clickhouse/client
+
+Create a client instance
+------------------------
```

Comment:
I would remove this entire section and show directly the code sample that works,
not showing the "just create a client, and here is the full documentation of the client".